server/database/config.js

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
@musonant This should be Db.js

Reply:
Oh...
I have a separate file, db.js that is responsible just for connecting the database. This config.js handles migration and seeding as well. Which is why I named it so. Is there a more suitable name I should give it? @tosmak16